### PR TITLE
Add Supply Chain demo

### PR DESCRIPTION
**Pull request by FlorianJacta**
image

image

image

[Supply Chain Optimization - Taipy.zip](https://github.com/user-attachments/files/16413895/Supply.Chain.Optimization.-.Taipy.zip)

### CODE REVIEW

File: docs/gallery/decision_support/2_supply_chain/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+title: Supply Chain Optimization
+category: decision_support
+data-keywords: optimization scenario cycle comparison
```
Comment (AlexandreSajus) [inline docs/gallery/decision_support/2_supply_chain/index.md]: Everything looks very good! Just missing `enterprise` keyword here

**Review by AlexandreSajus — CHANGES_REQUESTED** [review]
Looks good! Just a missing `enterprise` keyword

File: docs/gallery/decision_support/2_supply_chain/index.md (hunk 1 above)
Comment (FlorianJacta) [reply docs/gallery/decision_support/2_supply_chain/index.md]: Yes, I didn't commit it. Done

**Review by AlexandreSajus — APPROVED** [review]
Looks good to me!

**Pull request merged**